src/main/java/com/lmax/disruptor/dsl/Disruptor.java

```diff
@@ -236,6 +236,24 @@ public void handleExceptionsWith(final ExceptionHandler<? super T> exceptionHand
     }
 
     /**
+     * <p>Specify an exception handler to be used for event handlers and worker pools created by this Disruptor.</p>
+     * <p>
+     * <p>The exception handler will be used by existing and future event handlers and worker pools created by this Disruptor instance.</p>
+     *
+     * @param exceptionHandler the exception handler to use.
+     */
+    @SuppressWarnings("unchecked")
+    public void setDefaultExceptionHandler(final ExceptionHandler<? super T> exceptionHandler)
+    {
+        checkNotStarted();
+        if (!(this.exceptionHandler instanceof ExceptionHandlerWrapper))
+        {
+            throw new IllegalStateException("Mixing calls to handleExceptionsWith and setDefaultExceptionHandler is not supported.");
```

Comment:
We can do disruptor().setDefaultHandler().handleExceptionsWith(), so, maybe, not "mixing", but something like "callToDefaultExceptionHandler is not supported and useless after exceptionHandler has overridden by EventHandler specific ExceptionHandler using handleExceptionWith()"